Symfony/CS/ConfigInterface.php

```diff
@@ -105,4 +105,20 @@ public function usingCache();
      * @return bool
      */
     public function usingLinter();
+
+    /**
+     * Sets the path to the cache file.
+     *
+     * @param string $cacheFile The path to the cache file
+     *
+     * @return ConfigInterface The same instance
+     */
+    public function setCacheFile($cacheFile);
+
+    /**
+     * Returns the path to the cache file.
+     *
+     * @return string The path to the cache file
```

Comment:
I think we should remove them then. They are nearly identical to the method description.